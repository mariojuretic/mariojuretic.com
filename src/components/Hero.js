import React from 'react';
import { Link } from 'react-router-dom';

const Hero = () => (
  <section className="hero">
    <div className="content-container">
      <div className="hero__content">
        <h1 className="hero__title">Are you looking for a new website?</h1>
        <h3 className="hero__subtitle">I can build one for you! Check out my work.</h3>
        <Link to="/work" className="hero__button">My work</Link>
      </div>
    </div>
  </section>
);

export default Hero;