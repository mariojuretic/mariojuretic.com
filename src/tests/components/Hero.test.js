import React from 'react';
import { shallow } from 'enzyme';
import Hero from '../../components/Hero';

test('should render Hero correctly', () => {
  const wrapper = shallow(<Hero />);
  expect(wrapper).toMatchSnapshot();
});