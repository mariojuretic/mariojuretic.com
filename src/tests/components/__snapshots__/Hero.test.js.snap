// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Hero correctly 1`] = `
<section
  className="hero"
>
  <div
    className="content-container"
  >
    <div
      className="hero__content"
    >
      <h1
        className="hero__title"
      >
        Are you looking for a new website?
      </h1>
      <h3
        className="hero__subtitle"
      >
        I can build one for you! Check out my work.
      </h3>
      <Link
        className="hero__button"
        replace={false}
        to="/work"
      >
        My work
      </Link>
    </div>
  </div>
</section>
`;
